### PR TITLE
More 4.4 release notes updates

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: http://file.rdu.redhat.com/~choag/041620/rn-4.4-updates-3/release_notes/ocp-4-4-release-notes.html

### CODE REVIEW

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -123,6 +139,24 @@ console. This lets you view metrics that bring transparency to the
 [id="ocp-4-4-web-console"]
 === Web console
 
+[id="ocp-4-4-marketplace-integration-in-operator-hub"]
+==== IBM Marketplace integration in OperatorHub
+
+IBM Marketplace is now integrated in the OperatorHub, which is located in the
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:145]: "integrated in" or "integrated with?"

Is there a clearer word than "integrated?"

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -123,6 +139,24 @@ console. This lets you view metrics that bring transparency to the
...
+{product-title} web console.
+
+[id="ocp-4-4-edit-apps-in-topology-view"]
+==== Edit applications in the Topology view
+
+You can now edit applications from the Developer perspective by using the
+*Topology* view.
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:152]: Is this talking specifically about a UI control or the view itself? According to the repo guidelines, only bold the former.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -123,6 +139,24 @@ console. This lets you view metrics that bring transparency to the
...
+
+[id="ocp-4-4-create-helm-releases"]
+==== Create Helm releases
+
+You can now create Helm releases from the Helm charts provided in the *Developer
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:157]: Same as previous comment.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -760,3 +794,8 @@ URIs use the `git://` or `ssh://` scheme.
 the mirror registry only links to the builder service account. The pull secret
 must also link to the build config object.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1810904[*BZ#1810904*])
+
+* The service catalog cannot be enabled in {rh-openstack-first} 13 with Kuryr
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:798]: Consider:

> In {rh-openstack-first} 13 with Kuryr, if FIPS is disabled, you cannot enable the service catalog.

**Review by maxwelldb — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-4-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:152]: Only the view itself. I'll update both of these cases.

**Comment by codyhoag** [comment]
@maxwelldb thanks for reviewing! I just pushed those edits. Can you confirm?

File: release_notes/ocp-4-4-release-notes.adoc [hunk 5]
```diff
@@ -70,6 +70,23 @@ are just an example.
 See xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Installing a cluster on Azure using ARM templates]
 for details.
 
+[id="ocp-4-4-installing-cluster-on-openstack-upi"]
+==== Installing a cluster on OpenStack using user-provisioned infrastructure
+
+{product-title} 4.4 introduces support for installing a cluster on
+{rh-openstack-first} that runs on infrastructure that you provide].
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:77]: Now rendering to review... there's an extra `]` at the end of this sentence.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 6]
```diff
@@ -123,6 +140,25 @@ console. This lets you view metrics that bring transparency to the
...
+[id="ocp-4-4-edit-apps-in-topology-view"]
+==== Edit applications in the Topology view
+
+You can now edit applications from the Developer perspective by using the
+Topology view.
+
+[id="ocp-4-4-create-helm-releases"]
+==== Create Helm releases
+
+You can now create Helm releases from the Helm charts provided in the Developer
```
Comment (maxwelldb) [inline release_notes/ocp-4-4-release-notes.adoc:159]: `provided` -> `that are provided`

**Review by maxwelldb — CHANGES_REQUESTED** [review]
Couple more things.

**Pull request merged**